tests/unit_tests/PayplugTest.php

```diff
@@ -9,6 +10,18 @@
 class PayplugTest extends \PHPUnit_Framework_TestCase
 {
     public function testCanInitializeConfiguration()
+    {
+        Payplug::init(array(
+            'secretKey' => 'cba',
+            'apiVersion' => null
+        ));
+
+        $configuration = Payplug::getDefaultConfiguration();
+
+        $this->assertEquals('cba', $configuration->getToken());
```

Comment:
Tu peux ajouter ici l'utilisation de la valeur par défaut `$this->assertEquals('2019-06-14', $configuration->getApiVersion());`

Et un autre test qui vérifie que quand tu set une valeur tu as bien la bonne valeur stp :)

Reply:
J'ai crée la méthode `testCanInitializeDefaultConfiguration` qui check la version par défaut et la méthode `testCanInitializeConfiguration` qui test si lorsque l'on renseigne une date bidon, cette dernière est correctement set